dev-support/ci/pr_title_check.sh

```diff
@@ -42,6 +42,8 @@ assertMatch    '^HDDS-[1-9][0-9]{0,4}[^0-9]'       'Fail: Jira must be 1 to 5 di
 assertMatch    '^HDDS-[1-9][0-9]{0,4}\.'           'Fail: missing dot after Jira'
 assertMatch    '^HDDS-[1-9][0-9]{0,4}\. '          'Fail: missing space after Jira'
 assertNotMatch '[[:space:]]$'                      'Fail: trailing space'
+assertNotMatch '\.{3,}$'                           'Fail: trailing ellipsis indicates title is cut'
```

Comment:
Nit: Should this be length + trailing ellipsis? If not, the message can be that trailing ellipsis is not permitted.
The length (72) is the main cause here as it has prescriptive titles for commits.